src/helper/forward.h

```diff
@@ -24,6 +26,9 @@ class Request;
 
 } // namespace Helper
 
+using HelperPointer = RefCount<helper>;
+using StatefulHelperPointer = RefCount<statefulhelper>;
```

Comment:
Please avoid defining multiple different types referring to the same thing. Specifically the ```helper::Pointer``` and ```statefulhelper::Pointer``` types duplicate these in a confuse-able way.

FWIW, my rule of thumb is to prefer the class-scoped definitions whenever possible. However if the code needs global-scope types (as this helper code might), move everything to use those instead. So each class with a ```Pointer``` in public ```.h``` files only has one of the two styles. Type symbols defined locally in a ```.cc``` function do not count for this rule.